java/README.MD

```diff
@@ -1,24 +1,33 @@
-# Azure IoT Central ARM Java SDK samples
+# Azure IoT Central Java SDK Samples for Management APIs
 
+Sample code for using Azure IoT Central Management Plane APIs.
 Sample code for using [com.microsoft.azure.iotcentral.v2018_09_01:azure-mgmt-iotcentral](https://search.maven.org/artifact/com.microsoft.azure.iotcentral.v2018_09_01/azure-mgmt-iotcentral/1.0.0/jar) SDK provided by [Microsoft Azure](https://github.com/Azure). This sample code will check if the resource name is available, create or update an app, retrieve app information, update app, list all the apps that belongs to the specific resource group, and finally delete the app if uncommented.
 
 ## Get started
 
 ### Prerequisites
 - [Java JDK or JRE](https://www.oracle.com/technetwork/java/javase/downloads/index.html)
-- [Gradle](https://gradle.org/install/)
-- A resource group called **myResourceGroup** in your Azure subscription
+- [Maven](http://maven.apache.org/install.html)
```

Comment:
Do we want to keep the Gradle one since someone could be using Gradle instead of Maven? so have two side by side?

Reply:
The older code doesn't work with 2021 release - auth libraries have all been updated. So we should not keep it around (to prevent confusion). The gradle vs maven change is trivial for java developers, so I don't think it will add much value - we can always refer to it via past git commits. So if you are ok, I'd like to replace it instead of keeping the old code.